moira/db.py

```diff
@@ -1072,6 +1074,25 @@ def getEvents(self, trigger_id=None, start=0, end=100):
         defer.returnValue([anyjson.deserialize(e) for e in events])
 
     @defer.inlineCallbacks
+    @docstring_parameters(TARGETS)
+    def getTargets(self):
+        """
+        getTargets(self)
+
+        Returns all targets from db (variant = False)
+        or returns list targets in set TARGETS
+
+        :rtype: list
+        """
+        variant = False
+        if variant:
```

Comment:
It's temporal. Because for create set of metric name required to be moira/cache added it.